src/config/logger.ts

```diff
@@ -91,6 +91,11 @@ export const getLogger = (name: string): LoggerWithTarget => {
 	return logger.child({ name });
 };
 
+// This will log data to a restricted environment [env]-unsafe
+export const getUnsafeLogger = (name: string): LoggerWithTarget => {
+	return logger.child({ name, env_suffix: "unsafe" });
```

Comment:
this little fella `env_suffix: "unsafe"` is the difference, Splunk will index these logs off into there own env.

So on splunk you can search safe logging `env=stg-west` or unsafe logging `env="sg-west-unsafe"`

Reply:
What's the process of acquiring access to the unsafe logs?

Reply:
Still confirming that things will work.. but the intention is to have a ACL that will be controlled by Christian and is assigned to the unsafe environment on the microscopes observability page.... at this point it would be manual, but can look into other options so that access expires and we have better visibility of who used it.